src/css-rule-set-to-string.js

```diff
@@ -5,9 +5,14 @@ import camelCasePropsToDashCase from './camel-case-props-to-dash-case';
 import mapObject from './map-object';
 import {getPrefixedStyle} from './prefixer';
 
```

Comment:
We should probably follow convention here and make a `quote-value-if-needed` util file. Here is a similar util for reference: https://github.com/davidjbeveridge/radium/blob/2106d31209fd395ec1118478af87b4d7f06e3321/src/append-px-if-needed.js